src/main/java/com/google/sps/servlets/TablesFromDatabase.java

```diff
@@ -0,0 +1,56 @@
+
+package com.google.sps.servlets;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Enumeration;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns HTML that contains tables in the example database. */
+@WebServlet("/tables-from-db")
+public class TablesFromDatabase extends HttpServlet {
+
+  DatabaseClient dbClient;
+
+  public void init() {
+    Spanner spanner = SpannerOptions.newBuilder().build().getService();
+
+    //TODO: change "example-db" to the database that Millennia passes in from the db selection
```

Comment:
Note, we can do more here by hardcoding this in the javascript rather than the server. That way we don't need to change the server at all.

Reply:
Got rid of init() method